src/main/java/seedu/address/logic/commands/DeleteMeetingCommand.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.meeting.Meeting;
+
+/**
+ * Deletes a person identified using it's displayed index from the address book.
+ */
+public class DeleteMeetingCommand extends Command {
+
+    public static final String COMMAND_WORD = "delMeeting";
```

Comment:
use deletemeeting instead of delMeeting to standardize the commannds?

Reply:
Ok I've fixed this!